api/envoy/extensions/filters/http/gcp_authn/v3/gcp_authn.proto

```diff
@@ -40,6 +42,7 @@ message Audience {
 
 // Token Cache configuration.
 message TokenCacheConfig {
-  // Cache size. Default value is 0, which means the cache is disabled by default.
-  uint32 cache_size = 1;
+  // The number of cache entries.
+  // Default value 0 (i.e., proto3 defaults) disable the cache by default. Other default values will enable the cache.
+  google.protobuf.Int64Value cache_size = 1;
```

Comment:
Why `Int64Value` instead of `UInt64Value`?  Doesn't seem like a signed value would be very useful here.

Reply:
Thanks for the review! Sorry I didn't respond to your review comments along with commit push. I meant to wait for clean CI run first.

The reason i am using `Int64Value` is because the underlying [LRU cache implementation](https://source.corp.google.com/piper///depot/google3/third_party/jwt_verify_lib/simple_lru_cache/simple_lru_cache_inl.h;rcl=387869691;l=287) is using `int64_t`. I am not sure about the reason there but we cannot use the full range of unsigned anyway because of that.

Reply:
Friendly ping @markdroth

Reply:
Done. PTAL.Thanks!